<!-- http://wiki.ros.org/laser_scan_matcher -->

<launch>
   <node pkg="laser_scan_matcher" type="laser_scan_matcher_node" name="laser_scan_matcher_node" output="screen">
      <param name="base_frame"     value="laser" />
      <param name="fixed_frame"    value="odom" />
      <param name="use_imu"        value="false" />
      <param name="use_odom"       value="false" />
      <param name="use_vel"        value="false" />
      <param name="max_iterations" value="70" />
      <param name="publish_tf"     value="true" />
      <param name="publish_pose"   value="true" />
      <!--param name="clustering_threshold" value="0.5" /-->
   </node>
</launch>
